configs/default/RCTI-ARESF7.config

```diff
@@ -0,0 +1,119 @@
+# Betaflight / STM32F7X2 (S7X2) 4.2.0 Feb 17 2020 / 23:52:47 (d25c852098) MSP API: 1.43
+
+board_name ARESF7
+manufacturer_id RCTI
+
+# resources
+resource BEEPER 1 C15
+resource MOTOR 1 C06
+resource MOTOR 2 C07
+resource MOTOR 3 C08
+resource MOTOR 4 C09
+resource PPM 1 A09
+resource LED_STRIP 1 B00
+resource SERIAL_TX 1 A09
+resource SERIAL_TX 2 A02
+resource SERIAL_TX 3 C10
+resource SERIAL_TX 4 A00
+resource SERIAL_TX 5 C12
+resource SERIAL_RX 1 A10
+resource SERIAL_RX 2 A03
+resource SERIAL_RX 3 C11
+resource SERIAL_RX 4 A01
+resource SERIAL_RX 5 D02
+resource I2C_SCL 2 B10
+resource I2C_SDA 2 B11
+resource LED 1 C00
+resource SPI_SCK 1 A05
+resource SPI_SCK 2 B13
+resource SPI_SCK 3 B03
+resource SPI_MISO 1 A06
+resource SPI_MISO 2 B14
+resource SPI_MISO 3 B04
+resource SPI_MOSI 1 A07
+resource SPI_MOSI 2 B15
+resource SPI_MOSI 3 B05
+resource CAMERA_CONTROL 1 B06
+resource ADC_BATT 1 C02
+resource ADC_RSSI 1 C04
+resource ADC_CURR 1 C03
+resource SDCARD_CS 1 B12
+resource SDCARD_DETECT 1 C13
+resource PINIO 1 B01
+resource FLASH_CS 1 B12
+resource OSD_CS 1 A15
+resource GYRO_EXTI 1 C14
+resource GYRO_CS 1 A04
+resource USB_DETECT 1 C01
+
+# timer
+timer A09 AF1
+# pin A09: TIM1 CH2 (AF1)
+timer B00 AF3
+# pin B00: TIM8 CH2N (AF3)
+timer B06 AF2
+# pin B06: TIM4 CH1 (AF2)
+timer B07 AF2
+# pin B07: TIM4 CH2 (AF2)
+timer C06 AF2
+# pin C06: TIM3 CH1 (AF2)
+timer C07 AF2
+# pin C07: TIM3 CH2 (AF2)
+timer C08 AF2
+# pin C08: TIM3 CH3 (AF2)
+timer C09 AF2
+# pin C09: TIM3 CH4 (AF2)
+
+# dma
+dma ADC 1 0
+# ADC 1: DMA2 Stream 0 Channel 0
+dma pin A09 0
+# pin A09: DMA2 Stream 6 Channel 0
+dma pin B00 1
+# pin B00: DMA2 Stream 3 Channel 7
+dma pin B06 0
+# pin B06: DMA1 Stream 0 Channel 2
+dma pin B07 0
+# pin B07: DMA1 Stream 3 Channel 2
+dma pin C06 0
+# pin C06: DMA1 Stream 4 Channel 5
+dma pin C07 0
+# pin C07: DMA1 Stream 5 Channel 5
+dma pin C08 0
+# pin C08: DMA1 Stream 7 Channel 5
+dma pin C09 0
+# pin C09: DMA1 Stream 2 Channel 5
+
+# feature
+feature -RX_PARALLEL_PWM
+feature RX_SERIAL
+feature OSD
+
+# serial
+serial 0 64 115200 57600 0 115200
+serial 1 1024 115200 57600 0 115200
```

Comment:
I'm not sure this makes any sense. Enabling ESC sensor on this UART when the feature isn't even enabled.